### PR TITLE
veteran ranger buff pr (shotgun loadout)

### PR DESCRIPTION
**Pull request by lolman360**
## About The Pull Request

vet ranger shotgun loadout

## Why It's Good For The Game

req

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: vet ranger shotgun loadout
/:cl:

### CODE REVIEW

**Comment by Snoozer000** [comment]
Arent rangers suppose to have a focus on long range advantage? With an emphasis on revolvers and rifles? Wouldnt this negate legions close range advantage?

**Comment by AllCorruptingWyrm** [comment]
> 

a few staff seemed to be for it and note that in the games, vet rangers did have riot shotguns as well.
Compare it to the other loadouts, those being the amr (useful only if you join mid round,). the brush gun which is two .45-75 guns, and the gunslinger which breaks lore with no seqoia.

File: code/modules/jobs/job_types/ncr.dm [hunk 1]
```diff
@@ -454,6 +454,7 @@ Weapons		Service Rifle, Rockwell, 9mm pistol, all good.
 		/datum/outfit/loadout/vrclassic, // AMR, Sequoia
 		/datum/outfit/loadout/vrlite, // Brush Gun, Sequoia
 		/datum/outfit/loadout/vrcqc, // 2 x .45 Long colt revolvers
+		/datum/outfit/loadout/shotter //Riot shotty, Sequoia
```
Comment (OneOneThreeEight) [inline code/modules/jobs/job_types/ncr.dm:457]: /datum/outfit/loadout/shotgunner

File: code/modules/jobs/job_types/ncr.dm [hunk 2]
```diff
@@ -504,6 +505,16 @@ Weapons		Service Rifle, Rockwell, 9mm pistol, all good.
 		/obj/item/ammo_box/c4570 = 1,
 		)
 
+/datum/outfit/loadout/shotter
```
Comment (OneOneThreeEight) [inline code/modules/jobs/job_types/ncr.dm:508]: /datum/outfit/loadout/shotgunner

File: code/modules/jobs/job_types/ncr.dm [hunk 3]
```diff
@@ -504,6 +505,16 @@ Weapons		Service Rifle, Rockwell, 9mm pistol, all good.
...
+	name = "Veteran Shotter"
```
Comment (OneOneThreeEight) [inline code/modules/jobs/job_types/ncr.dm:509]: name = "Veteran Ranger Shotgunner"

**Review by OneOneThreeEight — CHANGES_REQUESTED** [review]

File: code/modules/jobs/job_types/ncr.dm [hunk 4]
```diff
@@ -504,6 +505,16 @@ Weapons		Service Rifle, Rockwell, 9mm pistol, all good.
...
 		)
 
+/datum/outfit/loadout/vrshotgunner
+	name = "Veteran Ranger Shotgunner"
+	suit_store = /obj/item/gun/ballistic/automatic/shotgun/riot
+	backpack_contents = list(
+		/obj/item/ammo_box/shotgun/buck = 2,
+		/obj/item/ammo_box/shotgun/slug = 1,
+		/obj/item/gun/ballistic/revolver/sequoia = 1,
+		/obj/item/ammo_box/c4570 = 1,
```
Comment (Looreydev) [inline code/modules/jobs/job_types/ncr.dm:515]: The comma isn't needed at the end of lists but that's just a next-level petty complaint from me at this point lol
```suggestion
		/obj/item/ammo_box/c4570 = 1
```

**Pull request merged**